tests/integration/test_charm.py

```diff
@@ -51,17 +52,23 @@ async def test_build_and_deploy(ops_test: OpsTest, local_charm: str | Path) -> N
         config={"plugin_pg_trgm_enable": "True", "plugin_btree_gin_enable": "True"},
     )
 
+    await ops_test.model.deploy(
+        CA_APP,
+        channel="latest/stable",
```

Comment:
1/stable is more recent (see https://charmhub.io/self-signed-certificates), could we use it instead?